libraries/HTTPClient/src/HTTPClient.h

```diff
@@ -304,8 +306,7 @@ class HTTPClient {
   String _acceptEncoding = "identity;q=1,chunked;q=0.1,*;q=0";
 
   /// Response handling
-  RequestArgument *_currentHeaders = nullptr;
-  size_t _headerKeysCount = 0;
+  std::vector<RequestArgument> _currentHeaders;
```

Comment:
Why, vector probably needs more bytes.

Reply:
Blame says that `#include <vector>` was added 3 years ago to HTTPClient.h for supporting Coockies.
I see no issue here.